dissector/v2g.lua

```diff
@@ -12,212 +12,245 @@ local V2GTP_HEADER_LENGTH = 8
 
 -- SDP (SECC Discovery Protocol)
 local SDP = {
-    -- header (8 bytes)
-    version = ProtoField.uint8("sdp.ProtocolVersion", "SDP Protocol Version", base.HEX),
-    version_inverted = ProtoField.uint8("sdp.InvertedProtocolVersion", "SDP Inverted Protocol Version", base.HEX),
-    payload_type = ProtoField.uint16("sdp.PayloadType", "SDP Payload Type", base.HEX),
-    payload_length = ProtoField.uint32("sdp.PayloadLength", "SDP Payload Length", base.HEX),
-
-    -- request payload (2 bytes only for normal SDP)
-    request_security = ProtoField.uint8("sdp.RequestSecurity", "SDP Request Security", base.HEX),
-    request_transport_proto = ProtoField.uint8("sdp.RequestTransportProtocol", "SDP Request Transport Protocol", base.HEX),
-    request_emsp_ids = ProtoField.string("sdp.RequestEMSPIDs", "EMSP IDs"),
-
-    -- response payload
-    response_secc_ip_addr = ProtoField.ipv6("sdp.ResponseSeccAddr", "SDP Response IP Address", base.HEX),
-    response_secc_port = ProtoField.uint16("sdp.ResponseSeccPort", "SDP Response Port", base.DEC),
-    response_security = ProtoField.uint8("sdp.ResponseSecurity", "SDP Response Security", base.HEX),
-    response_transport_proto = ProtoField.uint8("sdp.ResponseTransportProtocol", "SDP Response Transport Protocol", base.HEX),
-    response_emsp_ids = ProtoField.string("sdp.ResponseEMSPIDs", "EMSP IDs"),
+	-- header (8 bytes)
+	version = ProtoField.uint8("sdp.ProtocolVersion", "SDP Protocol Version", base.HEX),
+	version_inverted = ProtoField.uint8("sdp.InvertedProtocolVersion", "SDP Inverted Protocol Version", base.HEX),
+	payload_type = ProtoField.uint16("sdp.PayloadType", "SDP Payload Type", base.HEX),
+	payload_length = ProtoField.uint32("sdp.PayloadLength", "SDP Payload Length", base.HEX),
```

Comment:
It feels unnatural to display payload length in HEX.
```suggestion
	payload_length = ProtoField.uint32("sdp.PayloadLength", "SDP Payload Length", base.DEC),
```